app/controllers/api/v1x0/actions_controller.rb

```diff
@@ -1,61 +1,31 @@
 module Api
   module V1x0
     class ActionsController < ApplicationController
-      include Api::V1x0::Mixins::IndexMixin
-      include Api::V1x0::Mixins::RBACMixin
-
-      ADMIN_OPERATIONS     = [Action::MEMO_OPERATION, Action::APPROVE_OPERATION, Action::DENY_OPERATION, Action::CANCEL_OPERATION].freeze
-      APPROVER_OPERATIONS  = [Action::MEMO_OPERATION, Action::APPROVE_OPERATION, Action::DENY_OPERATION].freeze
-      REQUESTER_OPERATIONS = [Action::CANCEL_OPERATION].freeze
-
-      before_action :read_access_check, :only => %i[show]
-      before_action :create_access_check, :validate_create_action, :only => %i[create]
+      include Mixins::IndexMixin
 
       def index
         req = Request.find(params.require(:request_id))
+        authorize(req, :policy_class => RequestPolicy)
```

Comment:
why do you need to `authorize` and `policy_scope` together? shouldn't `policy_scope` already do permission check?

Reply:
`authorize` checks on `Request`, `policy_scope` checks on `Action` collections